src/screens/ContactScreen/index.js

```diff
@@ -195,9 +206,14 @@ const ContactScreen = ({navigation}) => {
     <SafeAreaView style={styles.container}>
       <StatusBar translucent={false} />
       <Header
-        title={StringConstant.chatTabHeaderCreateChatButtonText}
+        title={
+          sourceScreen === ANONYMOUS
+            ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+            : StringConstant.chatTabHeaderCreateChatButtonText
+        }
         containerStyle={styles.containerStyle}
         subTitle={'Next'}
+        subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(sourceScreen)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The title of the header component is now determined by whether the source screen is `ANONYMOUS` or not. The subtitle style also depends on whether there are any selected users and the source screen. Ensure that the `StringConstant.chatTabHeaderCreateAnonChatButtonText` and `StringConstant.chatTabHeaderCreateChatButtonText` constants contain appropriate text for the respective scenarios.
```diff
+ title={
+   sourceScreen === ANONYMOUS
+     ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+     : StringConstant.chatTabHeaderCreateChatButtonText
+ }
+ subTitle={'Next'}
+ subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(sourceScreen)}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->